src/passa/models/synchronizers.py

```diff
@@ -201,14 +234,18 @@ def sync(self):
 class Cleaner(object):
     """Helper class to clean packages not in a project's lock file.
     """
-    def __init__(self, project, default, develop):
+    def __init__(self, project, default, develop, sync=True):
```

Comment:
What does `sync` do? I am not sure Cleaner needs to be aware of Synchronizer.

Reply:
I added it for the moment to see how the CLI feels with it (think of it like a `--[no-]dry-run` option... The actual reason related to a need to short-circuit the test environment's cleaning functionality, so don't take this as me thinking it should stay... just as a kind of 'does this seem like a good idea?' check

Reply:
If it’s a dry run flag, I’d be more inclined to give it a different name, or (better IMO) to have a `DryRunCleaner` to do that.

Reply:
seems like a good plan, haven't revisited it yet